docs/dev/clusterversion.md

```diff
@@ -50,15 +50,32 @@ kind: DaemonSet
 metadata:
   name: cluster-network-operator
   namespace: openshift-cluster-network-operator
-$ cat <<EOF >version-patch.yaml
-> - op: add
->   path: /spec/overrides/-
->   value:
->     kind: DaemonSet
->     name: cluster-network-operator
->     namespace: openshift-cluster-network-operator
->     unmanaged: true
-> EOF
+```
+If there are currently no other overrides configured:
+```console
+$ cat <<EOF >version-patch-first-override.yaml
+- op: add
+  path: /spec/overrides
+  value:
+  - kind: DaemonSet
+    name: cluster-network-operator
+    namespace: openshift-cluster-network-operator
+    unmanaged: true
+EOF
+```
+To add to list of already existing overrides:
+```console
+$ cat <<EOF >version-patch-add-override.yaml
+- op: add
+  path: /spec/overrides/-
+  value:
+    kind: DaemonSet
+    name: cluster-network-operator
+    namespace: openshift-cluster-network-operator
+    unmanaged: true
+EOF
+```
+```console
```

Comment:
From my experience, this documentation will lead a user to successfully adding unmanaged objects.  This is best documentation I can do.  Note, when adding an element to already existing array, no hyphen before `kind` and add the `/-` to path.  Both are necessary.  When adding the overrides array, you need the hyphen and can't provide the `/-` to path. The path I understand, but the hyphen? :woman_shrugging: